app/pages/Auth/AuthPage.presenter/AuthPage.presenter.tsx

```diff
@@ -220,15 +228,12 @@ const AuthPage: FC = () => {
     }
   };
 
+  // TODO: improve error handling
   const onClickImport = async (accountName: string, entropy: string) => {
-    try {
-      if (isHex64(entropy)) {
-        await importLegacyAccount(accountName, entropy);
-      } else {
-        await importAccount(accountName, entropy);
-      }
-    } catch (err) {
-      /* nothing now... TODO: fix! */
```

Comment:
catching swallows errors, and prevents error propagation. removing to surface any errors being thrown here.